R/data.table.R

```diff
@@ -1202,6 +1231,17 @@ chmatch2 <- function(x, table, nomatch=NA_integer_) {
         xcols = w
         xcolsAns = seq_along(ansvars)
         icols = icolsAns = integer()
+      } else if (is.null(leftcols)) {
+        xcols = w[!wna]
+        xcolsAns = which(!wna)
+        ivars = names(i)
+        w2 = chmatch(ansvars[wna], ivars)
+        if (any(w2na <- is.na(w2))) {
+          ivars = paste("i.",ivars,sep="")
```

Comment:
According to codecov, these two lines are missing coverage causing diff coverage to be 94.28%. That's pretty good to be so few! Is there an easy way to cover them please? Diff coverage needs to be 100% unless there's a good reason; e.g. unusual stop()'s that are difficult to simulate to trigger.